### PR TITLE
Issue/40 make readme

### PR DESCRIPTION
**Pull request by Sebastiaanvm**
Feel free to add or change something.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,31 @@
 # frontend
-Frontend for psovea
+This is the frontend of psovea.
+
+# check out and go to another branch in github
+git checkout `<branch>`
```
Comment (emilhaugberg) [inline README.md:5]: Looks nicer to write multiple commands using three back ticks. 

```
command1
command2
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,31 @@
...
+example: git checkout master
+
+# Make a new branch.
+You can make new branches in github. Then use git checkout -b `<new branch>` in your terminal to go into that branch.
```
Comment (emilhaugberg) [inline README.md:9]: Maybe indicate the format of the branch `issue/<issue-number>-<description>`

File: README.md [hunk 3]
```diff
@@ -1,2 +1,31 @@
...
+
+# Start local development server
+Open the terminal in the frontend folder.
+Use "npm install" to install the libraries from the framework.
+Use "npm start" to start the server. The server will automatically refresh if you make a change in the code.
```
Comment (emilhaugberg) [inline README.md]: Use back ticks for npm commands

File: README.md [hunk 4]
```diff
@@ -1,2 +1,31 @@
...
+If you implement new libraries/packages you have to use "npm install" again.
+
+# File locations
+Most of the coded files are in the src folder. The code of all components are in de components folder.
+If you want to implement a new component, like a new type of graph, make a new folder in components. Use `<new component>`.js and `<new component>`.css files to implement that component.
```
Comment (emilhaugberg) [inline README.md:20]: Nice

**Review by emilhaugberg — COMMENTED** [review]
Looks good! I’ve written down some things that can be improved.

**Review by ericwoude — APPROVED** [review]

**Pull request merged**